Web/src/components/Navbar.jsx

```diff
@@ -18,12 +18,25 @@ import {
 } from '@chakra-ui/react';
 import { ColorModeSwitcher } from '../ColorModeSwitcher';
 import { FaSearch, FaBell, FaChevronDown } from 'react-icons/fa';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
+import supabase from "../supabaseClient";
+
 
 function Navbar() {
   const bgColor = useColorModeValue('white', 'gray.800');
   const textColor = useColorModeValue('gray.800', 'white');
```

Comment:
using hard coded colors isn't good here. ill keep this in for now but eventually we want to use themes from the theme provider as props to make it consistent around the app